content/rendering-with-batch/aws-fis/_index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "AWS Fault Injection Simulator"
+date: 2022-09-20T00:00:00Z
+weight: 150
+---
+
+## Overview
+
+[AWS Fault Injection Simulator (AWS FIS)](https://aws.amazon.com/fis/) is a fully managed service for running fault injection experiments on AWS that makes it easier to improve an application’s performance, observability, and resiliency. Fault injection experiments are used in chaos engineering, which is the practice of stressing an application in testing or production environments by creating disruptive events, such as sudden increase in CPU or memory consumption, observing how the system responds, and implementing improvements.
+
+This section of the lab will use AWS FIS to run a fault injection experiment in the ECS cluster associated to the AWS Batch Spot compute environment.
+
+## Features
```

Comment:
Can we shorten this section a bit? looks like a lot. You can include a link to check the detailed list of features in the AWS documentation